src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -26,39 +20,32 @@
 }
 
 .sf-toolbarreset {
-    position: fixed;
-    background-color: #f7f7f7;
+    background-color: #222;
+    box-shadow: 0 -1px 0px rgba(0, 0, 0, 0.2);
+    bottom: 0;
+    color: #EEE;
+    font: 11px Arial, sans-serif;
+    height: 36px;
     left: 0;
-    right: 0;
     margin: 0;
-    padding: 0 40px 0 0;
-    z-index: 6000000;
-    font: 11px Verdana, Arial, sans-serif;
+    padding: 0 36px 0 0;
+    position: fixed;
+    right: 0;
     text-align: left;
-    color: #2f2f2f;
-
-    background-image: -moz-linear-gradient(top, #e4e4e4, #ffffff);
-    background-image: -webkit-gradient(linear, 0% 0%, 0% 100%, from(#e4e4e4), to(#ffffff));
-    background-image: -o-linear-gradient(top, #e4e4e4, #ffffff);
-    background: linear-gradient(top, #e4e4e4, #ffffff);
-
-    bottom: 0;
-    border-top: 1px solid #bbb;
+    z-index: 6000000;
 }
 .sf-toolbarreset abbr {
-    border-bottom: 1px dotted #000000;
-    cursor: help;
-}
-.sf-toolbarreset span,
-.sf-toolbarreset div,
-.sf-toolbarreset td,
-.sf-toolbarreset th {
-    font-size: 11px;
+    border: none;
+    cursor: text;
```

Comment:
well, or at least don't reset the cursor at all, letting it become a pointer because of the abbr being inside a link. forcing ``cursor: pointer`` on all abbr is not good either in case they are not clickable